messages/setup.json

```diff
@@ -17,9 +17,9 @@
     "android:reqs:androidsdkprerequisitescheck:title" : "Checking Android SDK Prerequisites",
     "android:reqs:androidsdkprerequisitescheck:fulfilledMessage" : "Android SDK prerequisites are met.",
     "android:reqs:androidsdkprerequisitescheck:unfulfilledMessage" : "Android SDK is missing prerequisites: %s",
-    "android:reqs:androidhome:title" : "Checking ANDROID_HOME",
-    "android:reqs:androidhome:fulfilledMessage" : "ANDROID_HOME is set to %s.",
-    "android:reqs:androidhome:unfulfilledMessage" : "ANDROID_HOME not found. Setup ANDROID_HOME in your environment.",
+    "android:reqs:androidhome:title" : "Checking SDK root location",
+    "android:reqs:androidhome:fulfilledMessage" : "SDK root location is set to %s.",
+    "android:reqs:androidhome:unfulfilledMessage" : "SDK root location not found. Please set up ANDROID_SDK_ROOT or ANDROID_HOME in your environment.",
```

Comment:
🤔 I wonder if we should only recommend `ANDROID_SDK_ROOT` in this message, since `ANDROID_HOME` is deprecated? What do you think?

Reply:
I was thinking about that too, but then again on their documentation page they do mention that they still use ANDROID_HOME as default if it exists and prefer to use that over ANDROID_SDK_ROOT (and I don't understand why this is not the other way around if they indeed what to deprecate ANDROID_HOME). So I thought to leave the message as is here. But I can change it if you prefer.

Reply:
Yeah I didn't think about that, the fact that Google is still giving ANDROID_HOME precedence, if it's configured. That is strange. 🙂 I think that changes my mind, then. Let's just keep the current, until such a time as Google gets more aggressive about that deprecation.

Honestly, in our case the deprecation doesn't really matter that much anyway, since we're only leveraging the value by convention, not otherwise making use of its inherent value within the Android SDK.